### DISCUSSION TITLE
Greaseweazle V4 serial baud, parity, &c.?

### DISCUSSION THREAD

**Discussion by ssterling** [keirf/greaseweazle#174] [Q&A] [answered]
In short: I’ve got a bizarre setup which has led me to try and access a Greaseweazle V4 over a [ser2net](//ser2net.sourceforge.net/) connection.  However, I can’t figure out how to determine the baud rate, parity settings, &c. required as parameters for said ser2net connection to succeed.  Are they same for every Greaseweazle V4, and if not, is there a way for me to determine what they are?  My GW shows up as /dev/ttyACM0 and /dev/greaseweazle (installed udev rules), if that’s important.

**Comment by keirf** [comment, marked as answer]
Because there's not a real serial line backing the usb connection, actually none of the physical serial parameters matter and they will be ignored. A caveat to this is that several (non standard) baud rate values are used as control signals. Mainly to reset the Greaseweazle into a good state in case the normal data pipes get in a bad way.

**Comment by d235j** [comment]
You'd probably have more success with [USB/IP](http://usbip.sourceforge.net) instead, but I haven't tested it Linux to Linux and the Windows drivers have had issues in the past.
